server/src/main/java/org/jfrog/teamcity/server/global/DeployableArtifactoryServers.java

```diff
@@ -79,7 +79,9 @@ public List<String> getServerDeployableRepos(String serverUrlId, boolean overrid
             if (Objects.equals(serverUrlId, serverConfig.getId())) {
                 CredentialsBean deployingCredentials = CredentialsHelper.getPreferredDeployingCredentials(serverConfig, overrideDeployerCredentials, username, password);
                 try (ArtifactoryManager artifactoryManager = getArtifactoryManager(deployingCredentials, serverConfig)) {
-                    return artifactoryManager.getLocalRepositoriesKeys();
+                    List<String> localRepositories = artifactoryManager.getLocalRepositoriesKeys();
```

Comment:
localRepositories --> repos
(this list may include both local and federated repos).